docs/make.jl

```diff
@@ -6,7 +6,7 @@ makedocs(
          modules = [AbstractAlgebra],
          clean = true,
          doctest = true,
-         strict = false,
+         strict = true,
```

Comment:
Unfortunately it seems `strict` mode does not just abort upon errors (as its documentation says) but also for warnings as in https://github.com/Nemocas/AbstractAlgebra.jl/issues/707